src/test/codewhisperer/util/runtimeLanguageContext.test.ts

```diff
@@ -89,39 +91,166 @@ describe('runtimeLanguageContext', function () {
         })
     })
 
-    describe('mapVscLanguageToCodeWhispererLanguage', function () {
-        const cases = [
-            [undefined, undefined],
-            ['typescript', 'typescript'],
-            ['javascriptreact', 'jsx'],
-            ['typescriptreact', 'tsx'],
+    describe('normalizeLanguage', function () {
+        beforeEach(function () {
+            resetCodeWhispererGlobalVariables()
+        })
+
+        const codewhispererLanguageIds: CodewhispererLanguage[] = [
+            'c',
+            'cpp',
+            'csharp',
+            'go',
+            'java',
+            'javascript',
+            'jsx',
+            'kotlin',
+            'php',
+            'plaintext',
+            'python',
+            'ruby',
+            'rust',
+            'scala',
+            'sql',
+            'shell',
+            'tsx',
+            'typescript',
+        ]
+
+        for (const inputCwsprLanguageId of codewhispererLanguageIds) {
+            it(`should return itself if input language is codewhispererLanguageId - ${inputCwsprLanguageId}`, function () {
+                const actual = languageContext.normalizeLanguage(inputCwsprLanguageId)
+                assert.strictEqual(actual, inputCwsprLanguageId)
+            })
+        }
+
+        const platformLanguageIds: [PlatformLanguageId, CodewhispererLanguage][] = [
+            ['cpp', 'cpp'],
+            ['c_cpp', 'cpp'],
+            ['cpp', 'cpp'],
```

Comment:
this list seems redundant. is it expected that devs must update this list in the future?